app/usecase/user_interactor.go

```diff
@@ -14,6 +18,55 @@ type UserInteractor struct {
 	PermissionRepository PermissionRepository
 }
 
+func (interactor *UserInteractor) CheckUserPass(employeeId string, employeePass string) (token string, permissionLevel string, err error) {
+	//GENERATE TOCKEN IF EMPLOYEE INFO IS VALID
+	tokenId, err := interactor.UserRepository.IssueToken(employeeId, employeePass)
+	if err != nil {
+		return "", "", err
```

Comment:
Is it needed to return "", "", err? It can be only return.

Reply:
fixed!